src/content/docs/go/docs/plugins/google-genai.md

```diff
@@ -131,7 +131,7 @@ if err != nil {
 log.Println(resp.Text())
 ```
 
-See [Generating content with AI models](../models.md) for more information.
+See [Generating content with AI models](/go/docs/models.md) for more information.
```

Comment:
This goes to the raw .md file when clicked. https://genkit-dev-astro--pr-41-cx5ut7cs.web.app/go/docs/plugins/google-genai/

Should be just `/go/docs/model` I think

Reply:
good catch, fixed